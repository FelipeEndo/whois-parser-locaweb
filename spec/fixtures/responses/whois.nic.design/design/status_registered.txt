Domain Name: TOPLEVEL.DESIGN
Registry Domain ID: D44478444-CNIC
Registrar WHOIS Server: whois.porkbun.com
Registrar URL: https://porkbun.com/
Updated Date: 2021-08-30T22:42:53.0Z
Creation Date: 2017-02-16T20:11:45.0Z
Registry Expiry Date: 2022-02-16T23:59:59.0Z
Registrar: Porkbun, LLC
Registrar IANA ID: 1861
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Registrant Organization: Top Level Design LLC
Registrant State/Province: Oregon
Registrant Country: US
Registrant Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Admin Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Tech Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Name Server: NS-170.AWSDNS-21.COM
Name Server: NS-904.AWSDNS-49.NET
Name Server: NS-1067.AWSDNS-05.ORG
Name Server: NS-1873.AWSDNS-42.CO.UK
DNSSEC: unsigned
Billing Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registrar Abuse Contact Email: abuse@porkbun.com
Registrar Abuse Contact Phone: +1.5038508351
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-12-15T18:08:15.0Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

>>> IMPORTANT INFORMATION ABOUT THE DEPLOYMENT OF RDAP: please visit
https://www.centralnic.com/support/rdap <<<

The Whois and RDAP services are provided by CentralNic, and contain
information pertaining to Internet domain names registered by our
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com)

Access to the Whois and RDAP services is rate limited. For more
information, visit https://registrar-console.centralnic.com/pub/whois_guidance.
