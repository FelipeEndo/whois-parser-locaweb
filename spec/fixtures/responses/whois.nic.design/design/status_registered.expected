#disclaimer
  %s == "The Whois and RDAP services are provided by CentralNic, and contain information pertaining to Internet domain names registered by our our customers. By using this service you are agreeing (1) not to use any information presented here for any purpose other than determining ownership of domain names, (2) not to store or reproduce this data in any way, (3) not to use any high-volume, automated, electronic processes to obtain data from this service. Abuse of this service is monitored and actions in contravention of these terms will result in being permanently blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com) Access to the Whois and RDAP services is rate limited. For more information, visit https://registrar-console.centralnic.com/pub/whois_guidance."


#domain
  %s == "toplevel.design"

#domain_id
  %s == "D44478444-CNIC"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2017-02-16 20:11:45 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2021-08-30 22:42:53 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-02-16 23:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1861"
  %s.name         == "Porkbun, LLC"
  %s.url          == "https://porkbun.com/"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == nil
  %s[0].organization  == "Top Level Design LLC"
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == "Oregon"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns-170.awsdns-21.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns-904.awsdns-49.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns-1067.awsdns-05.org"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns-1873.awsdns-42.co.uk"
